intTests/test_llvm_global_fresh_pointer/test.saw

```diff
@@ -0,0 +1,29 @@
+enable_experimental;
+
+m <- llvm_load_module "test.bc";
+
+let foo_spec = do {
+  llvm_alloc_global "glb";
+  x_ptr <- llvm_fresh_pointer (llvm_int 32);
```

Comment:
It might be worth leaving a comment here stating that this is specifically intended to test the fact that `glb` and `x_ptr` are allowed to alias each other.